2022-02-11 13:54:15,242 - test_case01.py[line:21] - test_case01 - INFO - 用例序号:testcase_01
2022-02-11 13:54:15,242 - test_case01.py[line:22] - test_case01 - INFO - 用例名称:loginOauth
2022-02-11 13:54:15,242 - test_case01.py[line:23] - test_case01 - INFO - 请求url:http://uat.drp.orsd.tech/drpapi/oauth/loginOauth
2022-02-11 13:54:15,242 - test_case01.py[line:24] - test_case01 - INFO - 请求数据:{"username":"wuhaiyan", "password":"49ca2906599a55321586de19c9c835e1"}
2022-02-11 13:54:15,242 - test_case01.py[line:25] - test_case01 - INFO - 请求方式:POST
2022-02-11 13:54:15,242 - test_case01.py[line:26] - test_case01 - INFO - 请求头:header
2022-02-11 13:54:15,245 - test_case01.py[line:27] - test_case01 - INFO - 依赖:None
2022-02-11 13:54:15,245 - test_case01.py[line:28] - test_case01 - INFO - 预期结果:code
2022-02-11 13:54:15,371 - test_case01.py[line:30] - test_case01 - INFO - 返回结果为:{'code': '4001', 'msg': 'You have entered an incorrect password for five times. Please try again 962 minutes later.', 'data': None}
2022-02-11 13:54:15,371 - test_case01.py[line:31] - test_case01 - INFO - 返回状态码为:200
